### DISCUSSION TITLE
Support for the Soemtron Line Of Computers

### DISCUSSION THREAD

**Discussion by mmoeller86** [86Box/86Box#3333] [Ideas]
I request support for the Soemtron Line Of Computers.
They had either 80286, 80386 or 80486 Processors.

I read it on the main Robotron Page: https://www.robotrontechnik.de/index.htm?/html/computer/soemtronpc.htm

**Comment by OBattler** [comment]
We would need BIOS'es, motherboard photos, and documentation for that.
